docs/docs/index.md

```diff
@@ -30,6 +30,13 @@ If the MTA Build Tool encounters a module type in the `mta.yaml` file that is no
 
 All resource definitions are passed to the `mtad.yaml` file as is without mapping and validations (for example, if the resource is supported in the target platform or if the specified parameters match the type).
 
-### <b>Creating an MTA Archive According to the MTA Deploymnet Descriptor (`mtad.yaml` file)</b>
+### <b>Creating an MTA Archive According to the MTA Deployment Descriptor (`mtad.yaml` file)</b>
+
+The function is provided to assemble multiple modules, which are developed and prebuilt as separate projects, into one MTAR for deployment. The function is implemented by the `mbt assemble` command, which copies all modules, module required dependencies, module build artifacts, and resource definitions into an MTAR, according to the MTA Deployment Descriptor (mtad.yaml file).
+
+If a module doesn’t have the `path` attribute, the assemble process skips it. If a module has the `path` attribute, but its value is invalid, the assemble process fails. 
+
+If a required dependency or resource doesn’t have the `parameters.path` attribute, the assemble process skips it. If a required dependency or resource has the `parameters.path` attribute, but its value is invalid, the assemble process fails.
```

Comment:
If a required dependency or resource doesn’t have the `parameters.path` attribute, the assemble process skips it. If a required dependency or resource has the `parameters.path` attribute, but its value is invalid, the assemble process fails with the following error:
[Add sample error message that is displayed for required dependency or resource]